bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/TextInputImpl.java

```diff
@@ -70,6 +70,10 @@ public class TextInputImpl extends AbstractFieldImpl implements TextInput {
 
     /** End of Type number specific constraints **/
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
+    @Nullable
+    protected String displayValueExpression;
```

Comment:
Can you add this in `AbstractFieldImpl` and remove the usage from other places ? You could also refer on how `displayFormat` is done currently

Reply:
@rismehta but won't it appear in all components?

Reply:
Please review the implementation of the displayFormat. It shouldn't be included in the JSON unless explicitly set. This approach isn't correct. Please refrain from resolving comments; comments should be addressed by the reviewer.